jekyll/_cci2/install-machine-runner-3-on-docker.adoc

```diff
@@ -0,0 +1,110 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.4+
+---
+= Install machine runner 3.0 on Docker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions on how to install CircleCI's self-hosted machine runner (3.0) on Docker.
+:icons: font
+:experimental:
+:machine:
+:docker:
+
+This page describes how to install CircleCI's machine runner 3.0 with the Docker executor on server. If you are looking to set up self-hosted runners in a private Kubernetes cluster, visit the <<container-runner-installation#,Container runner>> page.
+
+[#new-recommended-method-container-runner]
+== Recommended method for Docker installation (container runner)
```

Comment:
Is this section to tell people that following this guide is NOT the recommended method?

Reply:
Yes exactly. This is really only a backup for customers who want to use a simple docker host setup rather than replicating cloud docker jobs on runner